tex/chapters/03-Theory.tex

```diff
@@ -93,25 +157,32 @@ \section{Fingerprint and Double Fingerprint}
 A \textbf{Fingerprint} $F_A(S, x, y)$ with $0 \leq x, y \leq 1$, $x+y \leq 1$ for strategy $S$ and \textbf{probe} $A$, is the function that returns the expected score of strategy $S$ against $\JA(A, x, y)$ for each possible $(x, y)$.
 \end{definition}
 
+% TODO I think you need to point out that $F_A$ is an analytical expression. So
+% a fingerprint is an analytical function. In a way that's very important when
+% it comes to probing with strategies that are not representable as FSM. You
+% should also include a discussion here about how Ashlock is constrained by
+% their choice of probe. Quote something they say about FSM reps.
 
 \begin{definition}\label{def:double-fingerprint}
 The \textbf{Double Fingerprint} $F_{AB}(S, x, y)$ with $0 \leq x, y \leq 1$ returns the expected score of strategy $S$ against $\JA(A, x, y)$ if $x+y \leq 1$, and $\JA(B, 1-y, 1-x)$ if $x+y \geq 1$.
 \end{definition}
 
-We now show how the double fingerprint, with an appropriate choice of probe, is equivalent to the fingerprint function over the unit square.
+$A$ is referred to as the \textbf{lower probe} and $B$ is referred to as the \textbf{upper probe}.
+This is shown in Figure \ref{fig:DualProbe}.
+It will now be shown how the double fingerprint, with an appropriate choice of lower and upper probe, is equivalent to the fingerprint function over the unit square.
```

Comment:
Reword this sentence.

In \cite{} it is shown that .... The proof in that paper will now be considered here.